docs/design/backup/#26-Admin-as-Superuser.txt

```diff
@@ -0,0 +1,11 @@
+title #26 Admin as Superuser
```

Comment:
There are commits so we can refer previous-updated things. Discard backups! Otherwise All Perfect!!